docs/guide-de/intro-yii.md

```diff
@@ -0,0 +1,61 @@
+Was ist Yii?
+============
+
+Yii ist ein leistungsstarkes, komponentenbasiertes PHP-Framework für die schnelle Entwicklung moderner Webanwendungen.
+Der Name Yii (ausgesprochen `Ji` oder `[ji:]`) bedeutet im Chinesischen "einfach und evolutionär". Es steht außerdem
+für das Akronym **Yes It Is!** (zu Deutsch **Ja, ist es!**).
+
+
+Wofür ist Yii am besten geeignet?
+---------------------------------
+
+Yii ist ein generisches Web-Entwicklungs-Framework, d.h. es kann für die Entwicklung aller Arten von Webanwendungen
+mit PHP verwendet werden. Aufgrund seiner komponentenbasierten Architektur und seines ausgefeilten Caching-Verfahrens
```

Comment:
Teile des Inhaltes von 12 und 13 stehen schon in Zeile 4. Könnte man eigentlich schön zusammenfassen.